src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationHandler.cs

```diff
@@ -253,5 +261,66 @@ protected override Task ApplyResponseChallengeAsync()
 
             return context.IsRequestCompleted;
         }
+
+        private string ComposeAuthorizationEndpoint(AuthenticationProperties properties) {
+            string endpointPath = AuthorizationEndpoint;
+
+            string endpoint =
+                !String.IsNullOrEmpty(Options.Endpoints.AuthorizationEndpoint) ?
+                Options.Endpoints.AuthorizationEndpoint :
+                ComposeEndpoint(properties, endpointPath);
+
+            // if AuthenticationProperties for this session specifies an environment property
+            // it should take precedence over the value in AuthenticationOptions
+            string environmentProperty = null; ;
```

Comment:
extra semicolon?

Reply:
Looks like a typo on my side, yes, thanks for catching it. Could you remove after merge or should I make the change and submit a new commit?

Reply:
I can remove it